scripts/gitlab/check_runtime.sh

```diff
@@ -24,7 +24,7 @@ SUBSTRATE_REPO_CARGO="git\+${SUBSTRATE_REPO}"
 SUBSTRATE_VERSIONS_FILE="bin/node/runtime/src/lib.rs"
 
 # figure out the latest release tag
-LATEST_TAG="$(git tag -l | grep -E '^v[0-9]+\.[0-9]+\.[0-9]+$' | sort -V | tail -n 1)"
+LATEST_TAG="$(git tag -l | grep -E '^v[0-9]+\.[0-9]+\.[0-9]-?[0-9]*+$' | sort -V | tail -n 1)"
```

Comment:
The regex ends with `[0-9]*+$`; either this is something I don't understand about grep regexes, or either the `*` or `+` is surplus.

Reply:
Yep that was a typo :) the PR's still WIP, I just opened it so that the test I'm fixing will get trigged on Gitlab